src/zoid/payment-fields/prerender.jsx

```diff
@@ -19,36 +19,13 @@ export function PaymentFieldsPrerender({ nonce } : PrerenderedPaymentFieldsProps
                             margin: 0;
                             width: 100%;
                             height: 100%;
-                            background: transparent;
+                            background: rgba(240,240,240,0.5);
```

Comment:
Is the merchant able to control the background color here when `transparent` is used?

Reply:
yes. background color is the page